src/Refactoring-Core/RBFindAndReplaceRefactoring.class.st

```diff
@@ -106,12 +108,18 @@ RBFindAndReplaceRefactoring >> findOccurrencesIn: rbMethod [
                       onSuccess: [ :map |
 	self extract: ((node start to: node stop) -> map) 
 	of: rbMethod.
+	occurrences := occurrences + 1.
 	flag := true.]
                       onFailure: [  ] .
 						flag ifTrue: [ self findOccurrencesIn: (rbMethod modelClass methodFor: rbMethod selector).
 							^ self]]].
 ]
 
+{ #category : #initialization }
+RBFindAndReplaceRefactoring >> initialize [ 
+	occurrences := 0.
```

Comment:
Hi @Lin777 please add a call to 
```
super initialize
```
Just in case somebody initialize something in the superclass (Now or in the future)

Reply:
Hi @tesonep, now understand why always call super initialize, even when that method is empty. Thanks